code-challenges.test.js

```diff
@@ -24,13 +30,37 @@
 
 
 // b) Create the function that makes the test pass.
-
-
+                    //PSUEDO
+//create function with a number argument
+//create empty array and iterate with stating inedx 2
+//push empty array into iterater and return.
+
+const favProblem = (nums) => { 
+    let fav = [0, 1];
+    let problem = [];
+    for(let i = 2; i <= nums; i++) {
+      fav[i] = fav[i - 1] + fav[i - 2]; 
+      problem.push(fav[i]);
+    }
+    
+    return problem;
+  }
+console.log(favProblem(6))
+console.log(favProblem(10))
+
+//this is obviously not my facorite problem.
```

Comment:
Did you solve this problem? DId you try anything different? What were the results of the different things you tried?